src/data/i18n/locales/json/es_ES.json

```diff
@@ -1 +1,3 @@
-{}
\ No newline at end of file
+{
+	"assetsFiltersDocuments": "Documento"
```

Comment:
My high-school español is a little rusty, but I've confirmed this translation in a spanish-english dictionary (and also, we aren't using this particular dialect). Just needed a change of some sort to exercise the jenkins job.